packages/graalvm/src/main/kotlin/elide/runtime/node/fs/NodeFilesystem.kt

```diff
@@ -41,13 +41,13 @@ import elide.runtime.exec.GuestExecutor
 import elide.runtime.exec.GuestExecutorProvider
 import elide.runtime.gvm.api.Intrinsic
 import elide.runtime.gvm.internals.intrinsics.js.AbstractNodeBuiltinModule
-import elide.runtime.gvm.internals.intrinsics.js.JsPromiseImpl.Companion.spawn
 import elide.runtime.gvm.js.JsError
 import elide.runtime.gvm.loader.ModuleInfo
 import elide.runtime.gvm.loader.ModuleRegistry
 import elide.runtime.interop.ReadOnlyProxyObject
 import elide.runtime.intrinsics.GuestIntrinsic.MutableIntrinsicBindings
 import elide.runtime.intrinsics.js.JsPromise
+import elide.runtime.intrinsics.js.JsPromise.Companion.of
```

Comment:
nit: mark `of` as `@JvmStatic` to avoid `.Companion.`

Reply:
I didn't know it avoided the import, that's a neat trick.